### ISSUE TITLE
5.5.0 container crashes on start

### ISSUE THREAD

**Issue by Scrumplex** [Hazmi35/jukebox#954]
Container crashes with error:

```
 /app/node_modules/thread-stream/index.js:209
       throw new Error('the worker has exited')
       ^
 Error: the worker has exited
     at ThreadStream.write (/app/node_modules/thread-stream/index.js:209:13)
     at Pino.write (/app/node_modules/pino/lib/proto.js:190:10)
     at Pino.LOG (/app/node_modules/pino/lib/tools.js:58:21)
     at Pino.fatal (/app/node_modules/pino/lib/levels.js:26:16)
     at process.<anonymous> (/app/dist/main.js:11:9)
     at process.emit (node:events:390:28)
     at process._fatalException (node:internal/process/execution:164:25)
```

I don't really have any special setup.

This is my .env:

```
# Use string ("" <==) if you need whitespace

# Secret things

# Your Discord bot's token
SECRET_DISCORD_TOKEN=hahalolxd

# Base config

# Your bot's prefix
CONFIG_PREFIX="$"

# A Discord ID(s), use comma if you want to add more than 1 owner
# NOTE: DO NOT ADD SOMEONE ELSE ID THAT YOU DON'T TRUST
CONFIG_OWNERS=138352803906191360

# Total shards to spawn, use "auto" or a number
# DEFAULT: AUTO
CONFIG_TOTALSHARDS=auto

# Should logging use debug level? Please note that this is not recommended in production
# INPUT: yes or no
# DEFAULT no
CONFIG_DEBUG=no

# Whether to cache users to memory
# INPUT: yes or no
# DEFAULT: no
CONFIG_CACHE_USERS=no

# Queue config

# Whether to allow users to add duplicate of tracks
# INPUT: yes or no
# DEFAULT: no
CONFIG_ALLOW_DUPLICATE=no

# Timeout in seconds before queue is deleted when no one is in the voice channel
# INPUT: Input should be in seconds
# DEFAULT: 180 (3 minutes)
CONFIG_DELETE_QUEUE_TIMEOUT=180

# Play command config

# Select track timeout when searching tracks
# INPUT: Input should be in seconds
# DEFAULT: 20 Seconds
CONFIG_SELECT_TIMEOUT=20

# Whether to disable selecting tracks on play command, this will automatically selects the first track from search results
# INPUT: yes or no
# DEFAULT: no
CONFIG_DISABLE_TRACK_SELECTION=no

# Maximum tracks to list in track selection (default: 10 max: 15)
# DEFAULT: 10
# MAX: 15
CONFIG_SEARCH_MAX_RESULTS=10

# Volume config

# Enable inline volume feature or not
# INPUT: yes or no
# DEFAULT: no
CONFIG_ENABLE_INLINE_VOLUME=no

# Default volume for music
# DEFAULT: 100
# NOTE: CONFIG_ENABLE_INLINE_VOLUME
CONFIG_DEFAULT_VOLUME=100

# Max volume allowed for music
# DEFAULT: 100
# NOTE: CONFIG_ENABLE_INLINE_VOLUME
CONFIG_MAX_VOLUME=100

# Music cache config

# Whether to download and cache tracks from youtube
# INPUT: yes or no
# DEFAULT: no
CONFIG_CACHE_YOUTUBE_DOWNLOADS=no

# Max track length in seconds that are allowed to be cached
# DEFAULT: 5400 (1 hour 30 minutes)
CONFIG_CACHE_MAX_LENGTH=5400

# Miscellaneous config

# Whether to disable the invite command or not
# INPUT: yes or no
# DEFAULT: no
CONFIG_DISABLE_INVITE_CMD=no

# What type of status does the bot use
# INPUT: playing, listening or watching
# DEFAULT: listening
CONFIG_STATUS_TYPE=listening

# What status does the bot display
# VARIABLES AVAILABLE: guildsCount, usersCount, playingCount, and botPrefix)
# DEFAULT: "music on {guildsCount} guilds!"
# NOTE ON {usersCount} VARIABLE: The value will be 0 if CONFIG_CACHE_USERS is not set to yes
CONFIG_STATUS_ACTIVITY="music on {guildsCount} guilds!"

# i18n config

# Language to be used in Jukebox, uses IETF language tags, for example: en-US, en-GB, id-ID
# Available languages: Please see the file list in folder src/langs, you can submit your language by contributing to Jukebox project at GitHub.
# DEFAULT: en-US
CONFIG_LANGUAGE=en-US


```

**Comment by Hazmi35** [comment]
Hello, are you running the container from root?

can you please sent the result from this commands:
`$ id`
`$ ls -la cache logs`

**Comment by Hazmi35** [comment]
I explained why this happens in #955

**Comment by Hazmi35** [comment]
Fixed in [Version 5.5.1](https://github.com/Hazmi35/jukebox/releases/tag/5.5.1).

Official Docker Image from Container Registries are being built in: [Build & Push to Docker Image Registry ](https://github.com/Hazmi35/jukebox/actions/runs/1703779356)
